plonky3/src/params/baby_bear.rs

```diff
@@ -58,17 +54,20 @@ const FRI_PROOF_OF_WORK_BITS: usize = 16;
 const RNG_SEED: u64 = 42;
 
 lazy_static! {
+    static ref ROUNDS: (usize, usize) = poseidon2_round_numbers_128::<Goldilocks>(WIDTH, D);
```

Comment:
Seems like `static ref (ROUNDS_F, ROUNDS_P): (usize, usize)` is not possible, not sure if there's a better way!